exampleAudioSink/README.md

```diff
@@ -0,0 +1,27 @@
+# Twilio Video AudioSink Example
+
+The project demonstrates how to use Twilio's Programmable Video SDK to access raw audio samples using the `AudioSink` API on `AudioTrack`.
+
+### Setup
+
+See the master [README](https://github.com/twilio/video-quickstart-android/blob/master/README.md) for instructions on how to generate access tokens and connect to a Room.
+
+This example requires Android Studio as well as two devices running Android API level 16 or higher.
+
+### Running
+
+Once you have setup your access token, install and run the example. You will be presented with the following screen:
+
+<kbd><img width="200px" src="../images/quickstart/audio_sink_launched.png"/></kbd>
```

Comment:
I need to get you an icon for this quickstart. However, I would just remove the image from the UI.

Reply:
Update the screenshot with the image view removed